.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
We've had a couple of PRs to `release`, shouldn't that get handled here as well?

Reply:
Yes, thank you. Fixed.